### DISCUSSION TITLE
멀티플레이 시도 중 SessionId를 불러오지 못합니다

### DISCUSSION THREAD

**Discussion by BumBall-Maker** [naverz/zepeto-studio-kor#1134] [5. Q&A]
![image](https://user-images.githubusercontent.com/87287709/183276512-5c16aff9-6a80-415c-ae95-8dd185319ecb.png)

![image](https://user-images.githubusercontent.com/87287709/183276522-d6bd025f-2b2d-4345-b169-103e92ab8f30.png)

session id의 타입 변환 문제인것 같긴 한데 정확한 오류 내용을 잘 모르겠습니다.
RoomCreated의 오류도 같은 문제인것 같은데 도움 부탁드립니다.
감사합니다. 

![image](https://user-images.githubusercontent.com/87287709/183276555-f668c821-5456-43d4-8a66-20c37ba54a01.png)

**Comment by YSHFM** [comment]
undefined로부터 RomCreated를 호출했다라는 에러 메시지로 보아서는 
![image](https://user-images.githubusercontent.com/106574541/183280680-b3fcbc90-c7a6-4938-b6ad-f066011994f0.png)
해당 이미지 처럼 먼저 제페토 멀티플레이가 인스펙터 내에 있는 지 확인이 필요해 보입니다.
